src/main/views/account-locked.njk

```diff
@@ -0,0 +1,9 @@
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ "Account locked" | safe }}
+{% endblock %}
+{% block content %}
+  <h1 class="govuk-heading-l">Your account has been locked</h1>
+  <p class="govuk-body">You have incorrectly entered the passcode 3 times and your account has been locked.</p>
```

Comment:
I think ' your account has been locked' needs to be changed to ' your account has now been locked' to match the screenshot

Reply:
not sure but the screenshot is saying 'your account has been locked'

Reply:
sorry i missed it. done now